doc/release/1.5.0-notes.rst

```diff
@@ -39,39 +57,230 @@ New features
+matrix
+
+Generalized QR factorization routines (``?geqrf``) now have full ``_lwork`` 
+counterparts.
+
+`scipy.linalg.cossin` Cosine Sine decomposition of unitary matrices has been 
+added.
+
+The function `scipy.linalg.khatri_rao`, which computes the Khatri-Rao product,
+was added.
+
+The new function `scipy.linalg.convolution_matrix` constructs the Toeplitz 
+matrix representing one-dimensional convolution.
 
 `scipy.ndimage` improvements
 ----------------------------
 
 
 `scipy.optimize` improvements
 -----------------------------
+The finite difference numerical differentiation used in various ``minimize``
+methods that use gradients has several new features:
+
+- 2-point, 3-point, or complex step finite differences can be used. Previously 
+  only a 2-step finite difference was available.
+- There is now the possibility to use a relative step size, previously only an
+  absolute step size was available.
+- If the ``minimize`` method uses bounds the numerical differentiation strictly 
+  obeys those limits.
+- The numerical differentiation machinery now makes use of a simple cache, 
+  which in some cases can reduce the number of function evaluations.
+
+There have been several improvements to `scipy.optimize.linprog`:
 
+- The ``linprog`` benchmark suite has been expanded considerably.
+- ``linprog``'s dense pivot-based redundancy removal routine and sparse 
+  presolve are faster
+- When ``scikit-sparse`` is available, solving sparse problems with 
+  ``method='interior-point'`` is faster
+
+The caching of values when optimizing a function returning both value and 
+gradient together has been improved, avoiding repeated function evaluations 
+when using a ``HessianApproximation`` such as ``BFGS``.
+
+``differential_evolution`` can now use the modern ``np.random.Generator`` as 
+well as the legacy ``np.random.RandomState`` as a seed.
 
 `scipy.signal` improvements
 ---------------------------
+A new optional argument ``include_nyquist`` is added to ``freqz`` functions in 
+this module. It is used for including the last frequency (Nyquist frequency).
 
+`scipy.signal.find_peaks_cwt` now accepts a ``window_size`` parameter for the 
+size of the window used to calculate the noise floor.
 
 `scipy.sparse` improvements
 ---------------------------
+Outer indexing is now faster when using a 2d column vector to select column 
+indices.
+
+`scipy.sparse.lil.tocsr` is faster
+
+Fixed/improved comparisons between pydata sparse arrays and sparse matrices
+
+BSR format sparse multiplication performance has been improved.
 
 `scipy.spatial` improvements
 ----------------------------
+`scipy.spatial.geometric_slerp` has been added to enable geometric 
+spherical linear interpolation on an n-sphere
+
+`scipy.spatial.SphericalVoronoi` now supports calculation of region areas in 2D 
+and 3D cases
+
+New ``cKDTree`` benchmarks are available for building and querying of balanced/
+unbalanced kd-trees.
```

Comment:
The benchmarks are nice, but the more interesting part is that the performance improved a lot for some cases - see gh-11653. That should be mentioned here.